### ISSUE TITLE
[Bug]: Unavailability of Aluminum in FTB University Version 1.3.0 for MC 1.16

### ISSUE THREAD

**Issue by dragonzero39** [FTBTeam/FTB-Modpack-Issues#325]
### Modpack

FTB University

### Modpack version

1.3.0

### Describe the bug

Aluminum is unavailable via any natural means after 3+ hours of searching and testing.

Bauxite Ore shows up in the JEI menu. Aluminum Ore only shows up in the creative inventory, and not in the JEI menu.

After several hours of mining, searching FTB wiki for alternative methods, and testing in Creative mode via spectator flying and creative digging, I was unable to find a single Bauxite or Aluminum Ore vein. Also attempted a full reinstall of the Modpack. Both times there were no issues during installation.

The Aluminum ore becomes a key part of University task progression fairly quickly.

### Steps to reproduce

Unnecessary to reproduce, as error is in natural World Generation.

Would probably be easiest to clear all stone and liquids for a chunk using WE or other tool, and see if Aluminum or Bauxite is generating. https://ftbwiki.org/Aluminum_Ore_(Tinkers%27_Construct) - https://ftbwiki.org/Bauxite_Ore_(Immersive_Engineering)

### Expected behaviour

N/A

### Additional information

I can be contacted via discord at Dragonzero39#9961 for any questions or requests for files. Very excited to further progress into the Modpack.

**Comment by quietust** [comment]
Have you tried using the Excavator from Immersive Engineering (assuming it still exists in 1.16)?

**Comment by dragonzero39** [comment]
I have not. I'm highly unfamiliar with most tech mods which is why I chose
to start with University.

I'll see if I can find a way to set up an excavator when I get home tonight
if I have time. Thanks for the recommendation.

**Comment by dragonzero39** [comment]
> Have you tried using the Excavator from Immersive Engineering (assuming it still exists in 1.16)?

So I spent some time playing with the excavator tool, before even getting to that I had to figure out how to take core samples or use survey tools. As someone entirely new to engineering mods, this took some time.

At no point during any of the surveying did I find Bauxite or Aluminum. I checked roughly 20 chunks. 

Even if it were available, with the intent of the University pack to be as an introduction to engineering and technical mods, Aluminum being potentially walled behind an excavator would be a massive roadblock. Most newer players would take many days or a few weeks to build their first excavator. 

Aluminum is required early on in order to create the Mechanical Tinkering book for the Tinker's Construct 3 quests in University. I would figure a small amount is available by natural means prior to any larger engineering machines, in order to do some of the smaller tasks like Drawbridge upgrades and the Mechanical Tinkering Book.

Or maybe if the intention is Aluminum solely being available via an excavator, that Aluminum is rewarded in a decent quantity early in a quest to support the tasks.

Maybe someone with more experience than me can confirm weather or not the Bauxite or Aluminum generates naturally.
